packages/cleanup-bare-metal.sh

```diff
@@ -53,8 +53,11 @@ for runtimeClass in "${unusedRuntimeClasses[@]}"; do
 
   # Remove references from containerd config
   echo "Removing ${runtimeClass} from ${CONFIG} ..."
-  dasel delete --file "${CONFIG}" --indent 0 --read toml --write toml "plugins.io\.containerd\.grpc\.v1\.cri.containerd.runtimes.${runtimeClass}" 2>/dev/null
-  dasel delete --file "${CONFIG}" --indent 0 --read toml --write toml "proxy_plugins.${SNAPSHOTTER}-${runtimeClass}" 2>/dev/null
+  dasel delete --file "${CONFIG}" --indent 0 --read toml --write toml "plugins.io\.containerd\.grpc\.v1\.cri.containerd.runtimes.${runtimeClass}" 2>/dev/null || true
+  # Containerd config v3
+  dasel delete --file "${CONFIG}" --indent 0 --read toml --write toml "plugins.io\.containerd\.cri\.v1\.runtime.containerd.runtimes.${runtimeClass}" 2>/dev/null || true
```

Comment:
I wonder how we could detect a cleanup breakage here. The `|| true` is probably right, since we want this to be idempotent, but shouldn't we at least log the errors?